src/libraries/System.Runtime.Numerics/src/System/Number.BigInteger.cs

```diff
@@ -1342,7 +1342,7 @@ static virtual bool TryParseWholeBlocks(ReadOnlySpan<TChar> input, Span<uint> de
             Debug.Assert(destiniation.Length * TParser.DigitsPerBlock == input.Length);
             ref TChar lastWholeBlockStart = ref Unsafe.Add(ref MemoryMarshal.GetReference(input), input.Length - TParser.DigitsPerBlock);
 
-            for (int i = 0; i < destiniation.Length - 1; i++)
+            for (int i = 0; i < destiniation.Length; i++)
```

Comment:
not something to handle in this PR, but I noticed this is `destiniation` not `destination` 😆 
(we can handle it separately after this goes in to avoid making it harder to review)

Reply:
oh, never mind, this is net new code and I was looking at the wrong diff view

It'd be great to fix the minor type in this PR then.